erpnext/manufacturing/doctype/operation/operation.js

```diff
@@ -10,5 +10,12 @@ frappe.ui.form.on('Operation', {
 				}
 			};
 		});
+	},
+});
+frappe.ui.form.on('Sub Operation', {
+	operation: function(frm, cdt, cdn) {
+		let row = locals[cdt][cdn]
+		row.time_in_mins = 0
+		frm.refresh_field('sub_operations')
```

Comment:
Why this much code for setting default? Just edit child table doctype and put `0.0` as default value. 

To fix existing records, write a patch that changes all `NULL` value to 0.